docs/install/docker/windows-docker-ce.md

```diff
@@ -0,0 +1,144 @@
+---
+navTitle: Docker Engine on Windows
+meta:
+   description: Install Docker Engine (Docker CE) on Windows.
+---
```

Comment:
Adding long-tail keywords will be beneficial. I have suggested some, but feel free to update them as needed.

```suggestion
   tags: 
       - docker for windows
       - install docker on windows
       - install docker on windows with wsl2
       - running docker on windows 11
---
```